app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,76 @@
+class TweetsController < ApplicationController
+
+  get '/tweets/new' do # displays 'create tweet' form
+    if logged_in?
+      erb :'tweets/create_tweet'
+    else
+      redirect to '/login'
+    end
+  end
+
+  post '/tweets' do # created tweet is submitted to database
+    if params[:content] == "" # checks submission for content
+      redirect to '/tweets/new' # if none found redirects user to new tweet form
+    else
+      user = User.find_by_id(session[:user_id])
+      @tweet = Tweet.create(:content => params[:content], :user_id => user.id)
```

Comment:
Another way to do this is to make use of the `has_many` relationship on User:
```
@tweet = current_user.tweets.create(content: params[:content])
```
Both approaches are good, but in some cases you may find this option useful.